core-bundle/src/Resources/contao/library/Contao/Email.php

```diff
@@ -603,7 +603,7 @@ public function sendTo()
 			}
 		}
 
-		System::log($strMessage, __METHOD__, $this->strLogFile);
+		System::getContainer()->get('contao.monolog.logger')->log($this->strLogFile, $strMessage);
```

Comment:
this looks like a wrong service name. Also, what is `$this->strLogFile`?

Reply:
Good catch. Its leftover from the former logger implementation.

In fact, there is no simple change for this line since `$this->strLogFile` – and thus the `action` of the Contao context – is set by the newsletter bundle to a dynamic value `ContaoContext::NEWSLETTER . '_' . $objNewsletter->id`.

Therefor the logging has to be wired up manually at this point. I have changed this in https://github.com/contao/contao/pull/3702/commits/24d4871cad77e59068682a98a94e873067b03dfd